sdk/core/azure-core-tracing-opentelemetry/test/ut/azure_core_test.cpp

```diff
@@ -10,6 +14,17 @@ int main(int argc, char** argv)
   // End users need to decide if SIGPIPE should be ignored or not.
   signal(SIGPIPE, SIG_IGN);
 #endif
+  // Declare a signal handler to report unhandled exceptions on Windows - this is not needed for
+  // other
+// OS's as they will print the exception to stderr in their terminate() function.
```

Comment:
I am not sure what's happening with this comment, lol, in the GitHub view. You may want to fix it up manually.

It shouldn't have any indentation since it is not within any #if or other scope/block.

Please fix here/elsewhere it is duplicated.

Reply:
It should be indented 2 spaces since it is a comment inside a function declaration.